### ISSUE TITLE
duplicated from opal/opal - source maps not working with clean rails 4x install

### ISSUE THREAD

**Issue by catmando** [opal/opal-rails#37]
Started GET "/assets/spec/models/opal_spec.rb" for 67.240.177.41 at 2015-03-04 01:10:58 +0000

Sprockets::FileNotFound (couldn't find file 'spec/models/opal_spec.rb'):
  sprockets (2.12.3) lib/sprockets/base.rb:153:in `resolve'
  opal (0.7.0) lib/opal/sprockets/server.rb:48:in `call'
  rack (1.5.2) lib/rack/cascade.rb:33:in `block in call'
  rack (1.5.2) lib/rack/cascade.rb:24:in `each'
  rack (1.5.2) lib/rack/cascade.rb:24:in `call'

**Comment by catmando** [comment]
just tried this out on a brand new clean VM, still no joy.
````
Started GET "/assets/spec/models/test_spec.map" for 67.240.177.41 at 2015-03-04 01:57:23 +0000


Started GET "/assets/opal.map" for 184.9.56.158 at 2015-03-04 01:57:23 +0000


Started GET "/assets/spec/models/test_spec.rb" for 67.240.177.41 at 2015-03-04 01:57:35 +0000

Sprockets::FileNotFound (couldn't find file 'spec/models/test_spec.rb'):
  sprockets (2.12.3) lib/sprockets/base.rb:153:in `resolve'
  opal (0.7.1) lib/opal/sprockets/server.rb:48:in `call'
  rack (1.5.2) lib/rack/cascade.rb:33:in `block in call'
  rack (1.5.2) lib/rack/cascade.rb:24:in `each'
  rack (1.5.2) lib/rack/cascade.rb:24:in `call'
  actionpack (4.1.6) lib/action_dispatch/journey/router.rb:73:in `block in call'
  actionpack (4.1.6) lib/action_dispatch/journey/router.rb:59:in `each'
  actionpack (4.1.6) lib/action_dispatch/journey/router.rb:59:in `call'
  actionpack (4.1.6) lib/action_dispatch/routing/route_set.rb:678:in `call'
  rack (1.5.2) lib/rack/etag.rb:23:in `call'
  rack (1.5.2) lib/rack/conditionalget.rb:25:in `call'
  rack (1.5.2) lib/rack/head.rb:11:in `call'
  actionpack (4.1.6) lib/action_dispatch/middleware/params_parser.rb:27:in `call'
  actionpack (4.1.6) lib/action_dispatch/middleware/flash.rb:254:in `call'
  rack (1.5.2) lib/rack/session/abstract/id.rb:225:in `context'
  rack (1.5.2) lib/rack/session/abstract/id.rb:220:in `call'
````

Here is the gem listing:

````
Using rake 10.4.2
Using i18n 0.7.0
Using json 1.8.2
Using minitest 5.5.1
Using thread_safe 0.3.4
Using tzinfo 1.2.2
Using activesupport 4.1.6
Using builder 3.2.2
Using erubis 2.7.0
Using actionview 4.1.6
Using rack 1.5.2
Using rack-test 0.6.3
Using actionpack 4.1.6
Using mime-types 2.4.3
Using mail 2.6.3
Using actionmailer 4.1.6
Using activemodel 4.1.6
Using arel 5.0.1.20140414130214
Using activerecord 4.1.6
Using bundler 1.7.6
Using coffee-script-source 1.8.0
Using execjs 2.2.2
Using coffee-script 2.3.0
Using thor 0.19.1
Using railties 4.1.6
Using coffee-rails 4.0.1
Using hike 1.2.3
Using multi_json 1.10.1
Using jbuilder 2.2.6
Using jquery-rails 3.1.2
Using sourcemap 0.1.1
Using tilt 1.4.1
Using sprockets 2.12.3
Using opal 0.7.1
Using opal-activesupport 0.1.0
Using opal-jquery 0.3.0
Using opal-rspec 0.4.1
Installing sprockets-rails 2.2.4 (was 2.2.2)
Using rails 4.1.6
Using opal-rails 0.7.0 (was 0.7.1)
Using rdoc 4.2.0
Using sass 3.2.19
Using sass-rails 4.0.5
Using sdoc 0.4.1
Using spring 1.2.0
Using sqlite3 1.3.10
Using turbolinks 2.5.3
Using uglifier 2.7.0
````

**Comment by elia** [comment]
Should be fixed on master, for now I don't think it's worth going back and fix older (than upcomping 0.8) version.
